ssh/tailssh/tailssh.go

```diff
@@ -254,141 +254,142 @@ func (c *conn) vlogf(format string, args ...any) {
 	}
 }
 
-// isAuthorized walks through the action chain and returns nil if the connection
-// is authorized. If the connection is not authorized, it returns
-// errDenied. If the action chain resolution fails, it returns the
-// resolution error.
-func (c *conn) isAuthorized(ctx ssh.Context) error {
-	action := c.currentAction
-	for {
```

Comment:
Moved to [clientAuth](https://github.com/tailscale/tailscale/pull/14803/files#diff-3f7d7fbab09c188bb05cc9c2a319a87e7da2b93b5e806a669c5771cdd7d9f908R279).